Program_1.0.py

```diff
@@ -11,13 +15,56 @@
 client = MongoClient()
 db = client.countries
 
+host_name = 'localhost'
+post_number = 9901
+
+
+class WikiHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        self.send_response(200)
+        self.end_headers()
+        input_data = eval('{' + (self.path.replace('%22', '"').replace('%20', ' ').replace('%7D', ',').replace('%7B', ': '))[8::] + '}')
+        input_data_content = input_data['content']
+        print(downloading_content(input_data_content))
+
+
+def run():
+    server_address = ('localhost', 9901)
+    httpd = HTTPServer(server_address, WikiHandler)
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        pass
+    httpd.server_close()
+
+
+def key_words_from_content(content):
+    if content.find('tag') != -1:
+        country_name = re.search('country\((.*)\);', content).group(1)
+        return country_name
+    elif content.find('checkflag') != -1:
+        link_to_flag = re.search('checkflag\((.*)\)', content).group(1)
+        return link_to_flag
+    else:
+        country_name = re.search('country\((.*)\)', content).group(1)
+        return country_name
+
 
 def saving_country_to_db(input_country):
+    country_name = input_country
     country = wikipedia.WikipediaPage(input_country).summary
     country_data = {'name': country_name, 'Summary': country}
     db.countries.insert_one(country_data)
 
 
+def whole_country_summary(input_country):
+    if db.countries.find({'name': input_country}).count() == 0:
```

Comment:
z tych 4 linijek mógłbyś zrobić funkcję a la "inesrting_to_db albos coś

Reply:
albo should_input_country_to_db zwracjacych true/false

Reply:
wgl cała logika zapisywania do BD nie powinan dziać się tu tylko na samym początku (po przyjściu zapytania)

Reply:
to do poprawy dalej